src/mattack_actors.h

```diff
@@ -98,25 +98,26 @@ class gun_actor : public mattack_actor
         /*@{*/
         /** Turrets may need to expend moves targeting before firing on certain targets */
 
-        int targeting_cost = 100; /** moves consumed before first attack can be made */
+        int targeting_cost = 100; /** Moves consumed before first attack can be made */
 
-        bool require_targeting_player = true; /** by default always give player some warning */
```

Comment:
1. "Doubling" the line count is irrelevant when it's going from 15 - 30.
2. There's no good reason for bad punctuation/capitalization in comments, if you want to blame someone, blame the original author.
3. If the comments weren't appended to lines, it wouldn't be an issue.